controller_manager/test/test_controller_manager_srvs.cpp

```diff
@@ -188,6 +188,16 @@ TEST_F(TestControllerManagerSrvs, reload_controller_libraries_srv) {
   auto request =
     std::make_shared<controller_manager_msgs::srv::ReloadControllerLibraries::Request>();
 
+  // Create a lambda to store the cleanup state change
+  bool cleanup_called = false;
+  auto cleanup_lambda =
+    [&](
+    const rclcpp_lifecycle::State &)
```

Comment:
Why do you have an argument which is not used?

Reply:
It's part of the callback provided by `register_on_cleanup`, I removed the  arg name to avoid unused arg warnings.